### DISCUSSION TITLE
Downloading just one video or photo

### DISCUSSION THREAD

**Discussion by alpenomil** [gilesknap/gphotos-sync#438] [Q&A]
Great tool - long term user. Recently I wanted only one video file. So I put it into a single album called "A"

`~/.local/bin/gphotos-sync   --album "A" --max-threads 2 --progress  --log-level debug ./destination
`
but I downloads the entire library. Is there a proper way to do it? It is authenticated etc.
Thanks

**Comment by gilesknap** [comment]
Hi @alpenomil that option is supposed to only download the one album. It will have to index the whole library though.

If it is downloading everything then that is a bug - but this is one of the tests in the automated suite so I'm surprised if it is not working.

**Comment by alpenomil** [comment]
Later I realised that if I use just A without quotes it works

`~/.local/bin/gphotos-sync   --album A --max-threads 2 --progress  --log-level debug ./destination 
`
for an album name with spaces like "Christmas Celebrations" then it works OK with 'spaces and double quotes'.

May be a python parser problem?

**Comment by gilesknap** [comment]
That makes no sense to me. If you are on linux then bash would interpret the quotes and they would not change what gets sent to Python. Not sure what windows does with command line parsing though.

If what you saw happening was indexing then you would not see it the second time as the DB records the last sync time and only indexes newer files.
